doc/changes/changes-1.2.0.md

```diff
@@ -0,0 +1,42 @@
+# script-languages-release 1.2.0, released 2021-06-21
+
+Code name: Use initial release of [script-languages-container-tool](https://github.com/exasol/script-languages-container-tool)
+
+## Summary
+
+This release uses the first release of the script-languages-container-tool. It also updates selected Python libraries, including the Google Cloud API, Microsoft Azure SDK and AWS SDK. Besides, this release provides bug fixes, small improvements. 
+
+## Package Version Comparison to Release 1.1.0
+
+* [Comparison of standard-EXASOL-7.1.0 in 1.2.0 with standard-EXASOL-7.1.0 in 1.1.0](package_diffs/1.2.0/diff_standard-EXASOL-7.1.0_standard-EXASOL-7.1.0/README.md)
+* [Comparison of standard-EXASOL-7.0.0 in 1.2.0 with standard-EXASOL-7.0.0 in 1.1.0](package_diffs/1.2.0/diff_standard-EXASOL-7.0.0_standard-EXASOL-7.0.0/README.md)
+* [Comparison of standard-EXASOL-6.2.0 in 1.2.0 with standard-EXASOL-6.2.0 in 1.1.0](package_diffs/1.2.0/diff_standard-EXASOL-6.2.0_standard-EXASOL-6.2.0/README.md)
+
+## Script-Language-Container-Tool (Exaslct)
+
+This release uses version 0.1.0 of the container tool, all relevant changes can be found [here](https://github.com/exasol/script-languages-container-tool/releases/tag/0.1.0).
+
+
+## Bug Fixes
+
+  - \#65: ForkingPickler PicklingError under MacOSX with Python 3.8
```

Comment:
No it doesn't. I thought it was some convention to use the #. Alsom I would say keeping the # shows the meaning of the number.

Reply:
if you use the number `#65` without the `\` Github generate links to the issue

Reply:
Are you sure, for example [here](https://github.com/exasol/integration-test-docker-environment/blob/master/doc/changes/changes-0.3.0.md) it's shown as plain text.

I added the backslash because locally (IntelliJ) the # is interpreted as markdown headline, and breaks formatting.